pymedphys/_streamlit/apps/mudensity/_dicom.py

```diff
@@ -145,13 +147,43 @@ def dicom_input_method(  # pylint: disable = too-many-return-statements
     rt_plan_name = str(dicom_plan.RTPlanName)
     st.write(f"Plan Name: `{rt_plan_name}`")
 
+    session_state = session.session_state(device_strict=True)
+
     try:
         deliveries_all_fractions = pymedphys.Delivery.from_dicom(
-            dicom_plan, fraction_number="all"
+            dicom_plan, fraction_number="all", device_strict=session_state.device_strict
         )
     except AttributeError:
         st.write(_exceptions.WrongFileType("Does not appear to be a photon DICOM plan"))
         return {}
+    except NotSupportedError as e:
+        st.warning(
+            """While extracting the delivery information out of the
+            DICOM file the following error occurred
+            """
+        )
+
+        st.write(e)
+
+        st.write(
+            """You have the option to set `device_strict=False` as long
+            as you are willing to accept that this is removing a safety
+            check from this software and will likely result in ignoring
+            one or more of the collimation devices that are within the
+            provided DICOM file.
+            """
+        )
+
+        if st.button("Set `device_strict=False` for this session"):
+            session_state.device_strict = False
```

Comment:
This is where you absolutely must have a logging of someone choosing this value, with whatever identity is available and the timestamp.  Preferably in a separate audit, but at least in a log.
If identify is not available intrinsically, the user should have to identify themselves and have that in the audit/log (it's better than nothing, and demonstrates that you at least considered the situation, and if they leave the identify field blank, don't accept a setting to False for device_strict).

This is exactly the kind of thing that ends in tears

Reply:
Yup, fair. I wonder if MU Density should have a "user" field right at the beginning, that user can then be included within the report produced at the end, and if any override is applied, that can also be included within the PDF report.

I'll work on that in a separate PR and then once that's done come back to this PR and see if that can be used here.